internal/core/v1beta1/types.go

```diff
@@ -72,6 +72,7 @@ const (
 
 var (
 	statusCodeReasons = map[int]string{
+		BadRequestErrorCode:               "Bad Request: %s",
```

Comment:
For future reference, [error strings should be lower case and free of punctuation](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

Reply:
Thanks. I need to re-read that and effective go again